### PR TITLE
fix: once ovn-encap-csum is set false,it can't be reset to true

### PR DESCRIPTION
**Pull request by zcq98**
# Pull Request

- [x] Make sure you have followed [Kube-OVN Code Style](https://github.com/kubeovn/kube-ovn/blob/master/CODE_STYLE.md).

## What type of this PR

Examples of user facing changes:

- Bug fixes

## Which issue(s) this PR fixes

Fix: once ovn-encap-csum is set false,it can't be reset to true

### CODE REVIEW

**Comment by zcq98** [comment]
ovn-encap-csum只能单向设置为false，但是没有把它配置成true的逻辑

File: pkg/daemon/config.go [hunk 1]
```diff
@@ -422,3 +426,13 @@ func disableChecksum() error {
...
+
+func enableChecksum() error {
+	// #nosec G204
+	raw, err := exec.Command(
+		"ovs-vsctl", "remove", "open", ".", "external-ids", "ovn-encap-csum").CombinedOutput()
+	if err != nil {
+		return fmt.Errorf("failed to remove ovn-encap-csum, %s", string(raw))
+	}
+	return nil
+}
```
Comment (zhangzujian) [inline pkg/daemon/config.go:438]: 可以与 `disableChecksum` 合并成一个函数。

File: pkg/daemon/config.go [hunk 2]
```diff
@@ -417,22 +412,20 @@ func setEncapIP(ip string) error {
...
+			return fmt.Errorf("failed to remove ovn-encap-csum, %s", string(raw))
+		}
+	} else {
+		raw, err := exec.Command(
+			"ovs-vsctl", "set", "open", ".", "external-ids:ovn-encap-csum=false").CombinedOutput()
+		if err != nil {
+			return fmt.Errorf("failed to set ovn-encap-csum false, %s", string(raw))
+		}
 	}
 	return nil
```
Comment (zhangzujian) [inline pkg/daemon/config.go:430]:
```suggestion
	raw, err := exec.Command("ovs-vsctl", "set", "open", ".", fmt.Sprintf("external-ids:ovn-encap-csum=%v", encapChecksum)).CombinedOutput()
	if err != nil {
		return fmt.Errorf("failed to set ovn-encap-csum false, %s", string(raw))
	}
	return nil
```

File: pkg/daemon/config.go [hunk 3]
```diff
@@ -413,12 +412,20 @@ func setEncapIP(ip string) error {
...
+func setChecksum(encapChecksum bool) error {
 	// #nosec G204
-	raw, err := exec.Command(
-		"ovs-vsctl", "set", "open", ".", "external-ids:ovn-encap-csum=false").CombinedOutput()
-	if err != nil {
-		return fmt.Errorf("failed to set ovn-encap-csum, %s", string(raw))
+	if encapChecksum {
+		raw, err := exec.Command(
+			"ovs-vsctl", "remove", "open", ".", "external-ids", "ovn-encap-csum").CombinedOutput()
+		if err != nil {
```
Comment (bobz965) [inline pkg/daemon/config.go:420]: log err

File: pkg/daemon/config.go [hunk 4]
```diff
@@ -413,12 +412,20 @@ func setEncapIP(ip string) error {
...
+			return fmt.Errorf("failed to remove ovn-encap-csum, %s", string(raw))
+		}
+	} else {
+		raw, err := exec.Command(
+			"ovs-vsctl", "set", "open", ".", "external-ids:ovn-encap-csum=false").CombinedOutput()
+		if err != nil {
```
Comment (bobz965) [inline pkg/daemon/config.go:426]: log err

**Review by zhangzujian — APPROVED** [review]

**Pull request merged**